src/sagemaker/fw_utils.py

```diff
@@ -119,10 +123,15 @@ def _using_merged_images(region, framework, py_version, framework_version):
     is_gov_region = region in VALID_ACCOUNTS_BY_REGION
     is_py3 = py_version == "py3" or py_version is None
     is_merged_versions = _is_merged_versions(framework, framework_version)
+
     return (
         ((not is_gov_region) or region in ASIMOV_VALID_ACCOUNTS_BY_REGION)
         and is_merged_versions
-        and (is_py3 or _is_tf_14_or_later(framework, framework_version))
+        and (
+            is_py3
+            or _is_tf_14_or_later(framework, framework_version)
+            or _is_pt_12_or_later(framework, framework_version)
```

Comment:
not in scope here, but I think it would be cleaner to maintain a constant that keeps track of which frameworks have both Python 2 & 3 support versus just Python 3 rather than writing new methods for each framework. maybe let `is_merged_versions` take in the Python version as another parameter.